### DISCUSSION TITLE
I wonder if this project has matured to this point yet?

### DISCUSSION THREAD

**Discussion by jeffwcx** [kaleidawave/ezno#76] [Questions]
I want to build a general metadata extraction platform based on this project. In addition to generating metadata for TypeScript libraries, this platform mainly generates metadata for front-end framework paradigms such as React and Vue. I wonder if this project has matured to this point yet?

**Comment by kaleidawave** [comment]
Hey, glad you are interested in the project!

At this time it probably isn't ready for your project. While a lot [is implemented](https://github.com/kaleidawave/ezno/blob/main/checker/specification/specification.md), it probably doesn't cover enough for analysing current libraries. Be sure to follow https://github.com/kaleidawave/ezno/milestone/1 for when more TypeScript and JavaScript features are added.

Sounds like an interesting idea. If you have any thoughts about the type information/metadata you need for the platform, be sure to let it be known here #24

**Reply by jeffwcx** [reply]
Thank you for your response. I will continue to follow your project. Once it becomes available, I will create a prototype first.
